packages/firestore/src/local/target_cache.ts

```diff
@@ -79,74 +80,74 @@ export interface QueryCache {
+   * @param targetData A TargetData instance to put in the cache.
    */
-  addQueryData(
+  addTargetData(
     transaction: PersistenceTransaction,
-    queryData: QueryData
+    targetData: TargetData
   ): PersistencePromise<void>;
 
   /**
    * Updates an entry in the cache.
    *
-   * The cache key is extracted from `queryData.query`. The entry must already
+   * The cache key is extracted from `targetData.target`. The entry must already
    * exist in the cache, and it will be replaced.
-   * @param {QueryData} queryData The QueryData to be replaced into the cache.
+   * @param {TargetData} targetData The TargetData to be replaced into the cache.
    */
-  updateQueryData(
+  updateTargetData(
     transaction: PersistenceTransaction,
-    queryData: QueryData
+    targetData: TargetData
   ): PersistencePromise<void>;
 
   /**
-   * Removes the cached entry for the given query data. It is an error to remove
-   * a query data that does not exist.
+   * Removes the cached entry for the given target data. It is an error to remove
+   * a target data that does not exist.
    *
    * Multi-Tab Note: This operation should only be called by the primary client.
    */
-  removeQueryData(
+  removeTargetData(
     transaction: PersistenceTransaction,
-    queryData: QueryData
+    targetData: TargetData
   ): PersistencePromise<void>;
 
   /**
    * The number of targets currently in the cache.
    */
   // Visible for testing.
-  getQueryCount(
+  getTargetCount(
     transaction: PersistenceTransaction
   ): PersistencePromise<number>;
 
   /**
-   * Looks up a QueryData entry by query.
+   * Looks up a TargetData entry by target.
    *
    * @param target The query target corresponding to the entry to look up.
-   * @return The cached QueryData entry, or null if the cache has no entry for
-   * the query.
+   * @return The cached TargetData entry, or null if the cache has no entry for
+   * the target.
    */
-  getQueryData(
+  getTargetData(
     transaction: PersistenceTransaction,
     target: Target
-  ): PersistencePromise<QueryData | null>;
+  ): PersistencePromise<TargetData | null>;
 
   /**
-   * Looks up a QueryData entry by target ID.
+   * Looks up a TargetData entry by target ID.
    *
-   * @param targetId The target ID of the QueryData entry to look up.
-   * @return The cached QueryData entry, or null if the cache has no entry for
-   * the query.
+   * @param targetId The target ID of the TargetData entry to look up.
+   * @return The cached TargetData entry, or null if the cache has no entry for
+   * the target.
    */
   // PORTING NOTE: Multi-tab only.
-  getQueryDataForTarget(
+  getTargetDataForTarget(
     txn: PersistenceTransaction,
     targetId: TargetId
-  ): PersistencePromise<QueryData | null>;
+  ): PersistencePromise<TargetData | null>;
 
   /**
-   * Adds the given document keys to cached query results of the given target
+   * Adds the given document keys to cached target query results of the given target
```

Comment:
nit: "cached target query results" seems a bit awkward.  I'd either stick with "cached query results" or simplify to just "cached results"  

And do the same for the following 2 diffs...